sentry/resource_sentry_project.go

```diff
@@ -274,6 +275,15 @@ func resourceSentryProjectUpdate(ctx context.Context, d *schema.ResourceData, me
 		params.ResolveAge = sentry.Int(v.(int))
 	}
 
+	if v, ok := d.GetOk("allowed_domains"); ok {
+		allowed_domains_set := v.(*schema.Set)
```

Comment:
snake_case is not Go style :-) Should be camelCase.

Reply:
Nice catch! Thank you! 😆 Goodness - my brain struggles writing in different programming languages sometimes. Fixed in https://github.com/Canva/terraform-provider-sentry/pull/37/commits/cd381ec12b6ae2c438d2b0bf1d0e7234fc499d5a